Resources/Textures/_NF/Objects/Decoration/Flora/flora_tree_palm.rsi/meta.json

```
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from goonstation at https://github.com/goonstation/goonstation/blob/master/icons/misc/beach2.dmi",
```

```suggestion
    "copyright": "Taken from goonstation at https://github.com/goonstation/goonstation/blob/4cd0cd164228c828cad0de0cbe5bffec597aac42/icons/misc/beach2.dmi",
```

Might as well tie it down to a specific version - sprite itself is old as the dirt and the artist would've been in a PR from a repo before it was on github.